python/ray/rllib/tuned_examples/pong-ppo.yaml

```diff
@@ -0,0 +1,12 @@
+pong-deterministic-dqn:
+    env: PongDeterministic-v4
+    run: PPO
+    resources:
+        cpu: 1
+        gpu: 1
+    stop:
+        episode_reward_mean: 21
+    config:
+        gamma: 0.99
+        num_sgd_iter: 20
+        devices: ["gpu:0"]
```

Comment:
should this have a leading "/"?

Reply:
Done.  Weirdly it worked before.  Aside: "/gpu" should work too since TF does prefix matching on device strings.